pkg/config/image.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+)
+
+type ImageConfig struct {
+	// EIB Metadata
```

Comment:
Such comments are unnecessary and might be misleading. Every comment above a field or struct definition serves as a documentation entry point i.e. if you open up the "documentation" for `ApiVersion` it will display this text.

I imagine we'll end up with nested structs for the metadata, base config, etc. anyway so getting rid of the comments is probably the best idea.

Reply:
Ahh, interesting. Good call though, you get my intention but I see how from the language standpoint it's the wrong way to achieve it.